drivers/storage/class/ramdisk/ramdisk.c

```diff
@@ -702,7 +702,7 @@ RamdiskCreateRamdisk(IN PDEVICE_OBJECT DeviceObject,
         Irp->IoStatus.Information = STATUS_SUCCESS;
     }
 
-    /* We're done */
+    /* We are done */
```

Comment:
Yeah I would argue that having a ton of unrelated changes like this is much worse @HBelusca.
@BieHDC no need for action here, I'll shut up about it.